docs/docs/50-running/04-volumes.md

```diff
@@ -30,14 +30,27 @@ The volume class used, including the default, may have restrictions on the size
 
 You can see a list of available volume classes and their restrictions, if any, with the [`acorn offerings volumeclasses`](100-reference/01-command-line/acorn_offerings_volumeclasses.md) command.
 
-## Using precreated volumes
+## Using pre-existing volumes
```

Comment:
Do we want to add something about not being able to use pre-existing Persistent Volumes created outside of Acorn?